nemo/collections/nlp/modules/common/megatron/megatron_utils.py

```diff
@@ -82,16 +87,19 @@ def get_megatron_lm_model(pretrained_model_name: str, config_file: Optional[str]
                 "num_layers": configf['num-layers'],
                 "max_position_embeddings": configf['max-seq-length'],
             }
+    elif config_dict:
```

Comment:
not supported, not tested

Reply:
Not supported? How is that different from reading from config file ? We do pass a dict down to actual Megatron constructor.